projects/1.18/assets/create-steam-n-rails/railways/lang/zh_cn.json

```
@@ -84,7 +213,24 @@
 	"railways.ponder.conductor_redstoning.text_3": "当玩家戴着与其颜色不一致的列车长帽时，指挥机器人不会激活红石元件",
 	
 	"railways.ponder.conductor_toolboxing.header": "使用指挥机器人运输工具箱",
-	"railways.ponder.conductor_toolboxing.text_1": "工具箱装备在指挥机器人身上时，其功能正常",
+	"railways.ponder.conductor_toolboxing.text_1": "工具箱装备在指挥机器人身上时，其功能可以正常使用",
```

这句话主语要不要中置？感觉这样子前置有点怪

当然不改也行

```suggestion
	"railways.ponder.conductor_toolboxing.text_1": "装备在指挥机器人身上时，工具箱的功能可以正常使用",
```